src/test/java/no/unit/alma/bibs/AlmaItemsClientTest.java

```diff
@@ -464,8 +503,8 @@ void testDeleteRequest() {
 
     @Test
     void testUpdateCommentOnRequest() {
-        final String COMMENT_FORMAT = "Ordernum: %s";
-        final String APPENDED_COMMENT_FORMAT = "Ordernum: comment\nOrdernum: %s";
+        final String commentFormat = "Ordernum: %s";
+        final String appendedCommentFormat = "Ordernum: comment\nOrdernum: %s";
```

Comment:
Det er ikke etter min behag, må jeg innrømme. En Final børde være capital letters. Jeg vet det kommer fra checkstyle (but I dont like it)